src/components/FilterSidebar/RangeSelector/RangeSelector.tsx

```diff
@@ -0,0 +1,70 @@
+import { useState } from "preact/hooks"
+import Icon from "@/elements/Icon/Icon"
+import RadioButton from "@/elements/RadioButton/RadioButton"
+import { SearchStatsFacet } from "@nosto/nosto-js/client"
+import { useRangeSelector } from "@nosto/search-js/preact/hooks"
+import styles from "./RangeSelector.module.css"
+import { cl } from "@nosto/search-js/utils"
+
+type Props = {
+  /** The stats facet to display range options for */
+  facet: SearchStatsFacet
+  /** Size of auto-generated buckets (default: 100) */
+  rangeSize?: number
+}
+
+export default function RangeSelector({ facet, rangeSize = 100 }: Props) {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const { min, max, range, ranges, updateRange, handleMinChange, handleMaxChange, isSelected } = useRangeSelector(
+    facet.id,
+    rangeSize
+  )
+  const [active, setActive] = useState(false)
```

Comment:
add a defaultActive prop that defaults to false and use it in the storybook file
apply the same change to the RangeFacet component